datamonster_api/lib/data_group.py

```diff
@@ -0,0 +1,133 @@
+from .base import BaseClass
+from .errors import DataMonsterError
+import numpy as np
+
+date_regex = r'\d{4}-\d{2}-\d{2}'
+
+
+class DataGroup(BaseClass):
+    """Representation of a DataGroup in DataMonster
+
+    :param _id: (int) unique internal identifier for the Data Group
+    :param name: (str) name of the Data Group
+    :param columns: (list of ``DataGroupColumn`` objects) representing columns of uploaded data
+    :param dm: ``DataMonster`` object
+    """
+
+    def __init__(self, _id, name, columns, dm):
+        self.id = _id
+        self.name = name
+        self.columns = columns
+        self.dm = dm
+
+    def __hash__(self):
+        return hash(self.id)
+
+    def __eq__(self, obj):
+        return isinstance(obj, DataGroup) and self.id == obj.id
+
```

Comment:
i noticed you have a repr for the other one, does this one not warrant one?

Reply:
The base class that DataGroup extends has a repr